app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,40 @@
 class BookingsController < ApplicationController
+<<<<<<< Updated upstream
+=======
+  def new
+    @booking = Booking.new
+    @theme = Theme.find(params[:theme_id])
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @theme = Theme.find(params[:theme_id])
+    @booking.theme = @theme
+    @booking.user = current_user
+
+
+    if @booking.save
+      redirect_to theme_path(@theme), notice: "Booking was successfully created."
+    else
+      render :new
+    end
+  end
+
+  def edit
+    @booking = Booking.find(params[:id])
+  end
+
+  def update
```

Comment:
Start adding @booking = Booking.find(params[:id]) in your update action because we did not set a before_action method

Reply:
You forgot to pass the booking_params as an argument of your @booking.update

Reply:
The changes are done.